### PR TITLE
[stable/chart] Added CRDs for Resolvers 

### PR DESCRIPTION
**Pull request by jamessthompson**
ConsulResolver, KubernetesEndpointResolver, and KuberetesServiceResolver

Signed-off-by: James Thompson <jst2120@gmail.com>

#### What this PR does / why we need it:
Adds the missing resolvers that are reported in the admin UI.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/ambassador/templates/crds.yaml [hunk 1]
```diff
@@ -28,6 +28,57 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
+metadata:
+  name: consulresolvers.getambassador.io
```
Comment (Flydiverny) [inline stable/ambassador/templates/crds.yaml:32]: Can you add
```yaml
  labels:
    app.kubernetes.io/name: {{ include "ambassador.name" . }}
    helm.sh/chart: {{ include "ambassador.chart" . }}
    app.kubernetes.io/instance: {{ .Release.Name }}
    app.kubernetes.io/managed-by: {{ .Release.Service }}
  {{ if .Values.crds.keep }}
  annotations:
    "helm.sh/resource-policy": keep
  {{ end }}
```

So it matches the other CRDs we have :)

**Review by Flydiverny — COMMENTED** [review]
Perhaps we can update the app version and image tag to 0.71.0 as well? As there's not much use of the new CRDs with the default image :)

File: stable/ambassador/templates/crds.yaml (hunk 1 above)
Comment (jamessthompson) [reply stable/ambassador/templates/crds.yaml:32]: Sorry, added.

File: stable/ambassador/values.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ securityContext:
 
 image:
   repository: quay.io/datawire/ambassador
-  tag: 0.70.1
+  tag: 0.71.0
```
Comment (Flydiverny) [inline stable/ambassador/values.yaml:49]: 👍 can you uppdate in Chart.yaml as well and README.md? :) (sucks but we need to update in 3 places 😞)

Comment (kflynn) [reply stable/ambassador/values.yaml:49]: I've been meaning to ask about this. Can we include scripting in this repo, or will that break things?

Comment (jamessthompson) [reply stable/ambassador/values.yaml:49]: Updated

**Comment by kflynn** [comment]
/lgtm

Thanks, @jamessthompson! 🙂

**Pull request merged**